docs/can-canjs/can-api.md

```diff
@@ -0,0 +1,289 @@
+@page api API Docs
+@parent canjs 2
+@outline 0
+@package ../../package.json
+@templateRender <% %>
+@description
+Welcome to the CanJS docs!
```

Comment:
This is shown when you hover over an item in the navigation, so I would add a little bit more text here about what’s in this section.